attributes/config.rb

```diff
@@ -0,0 +1,6 @@
+
+default['apache_zookeeper']['config_dir'] = '/opt/zookeeper/current/conf'
+default['apache_zookeeper']['bin_dir'] = '/opt/zookeeper/current/bin'
+default['apache_zookeeper']['data_dir'] =  '/var/opt/zookeeper/data'
```

Comment:
Similarly I would use the actual `dataDir` setting in `zoo.cfg` instead of also setting this here